### PR TITLE
CLI: Filter out internal static dir logging

### LINKED ISSUE #20938: [Bug]: Default fonts are not found using v7 beta
### Describe the bug

When using the beta branch with Vite, the fonts are not loaded:

image

### To Reproduce

```shell
https://stackblitz.com/github/storybookjs/sandboxes/tree/next/lit-vite/default-ts/after-storybook
```

### System

```shell
Environment Info:
- System (WSL):
  - OS: Linux 5.15 Alpine Linux
  - CPU: (12) x64 AMD Ryzen 5 3600 6-Core Processor
- Binaries:
  - Node: 18.12.1 - /usr/bin/node
  - Yarn: 3.3.1 - /usr/bin/yarn
  - npm: 9.1.2 - /usr/bin/npm
```

### Additional context

The error happens with the default template; no extra steps are needed to run the generated storybook files unchanged.

### Edits
- The issue seems to be happening on non-vite frameworks, so removed the Vite from the title.

### PR DESCRIPTION
**Pull request by ndelangen**
Closes https://github.com/storybookjs/storybook/issues/20938#issuecomment-1433435814

## What I did

Remove the log line in the command-line telling the user we've picked up their staticDir options.

### CODE REVIEW

File: code/lib/core-server/src/utils/server-statics.ts [hunk 1]
```diff
@@ -41,10 +41,7 @@ export async function useStatics(router: any, options: Options) {
                 directory: dir,
               })
             : dir;
-          const { staticDir, staticPath, targetEndpoint } = await parseStaticDir(relativeDir);
-          logger.info(
-            chalk`=> Serving static files from {cyan ${staticDir}} at {cyan ${targetEndpoint}}`
```
Comment (yannbf) [inline code/lib/core-server/src/utils/server-statics.ts:46]: I always thought this was quite useful, especially as a way to debug issues. If you feel really strong about removing them, can we at least change it to logger.debug?

Comment (shilman) [reply code/lib/core-server/src/utils/server-statics.ts:46]: I agree this is useful, as does the original commenter

Comment (ndelangen) [reply code/lib/core-server/src/utils/server-statics.ts:46]: OK, well if others find it useful, I'll add it back with a filter..

Doe y'all like this solution better?

File: code/lib/core-server/src/utils/server-statics.ts [hunk 2]
```diff
@@ -42,9 +42,14 @@ export async function useStatics(router: any, options: Options) {
               })
             : dir;
           const { staticDir, staticPath, targetEndpoint } = await parseStaticDir(relativeDir);
-          logger.info(
-            chalk`=> Serving static files from {cyan ${staticDir}} at {cyan ${targetEndpoint}}`
-          );
+
+          // Don't log for the internal static dir
+          if (!targetEndpoint.startsWith('sb-')) {
```
Comment (shilman) [inline code/lib/core-server/src/utils/server-statics.ts:47]:
```suggestion
          if (!targetEndpoint.startsWith('/sb-')) {
```

**Review by shilman — APPROVED** [review]

**Pull request merged**